examples/using-gatsby-image/src/pages/dominant-color.js

```diff
@@ -24,8 +24,8 @@ const DominantColor = ({ data, location }) => (
     <h2>Progressive Loading with Minimal Effort</h2>
     <p>
       The magic of Gatsby Image's Dominant color technique means that you can
-      load an image at moderate resolution and not have to bother with creating
-      a small thumbnail yourself. Gatsby Image will automatically create a
+      load a unique image placeholder and not have to bother with creating a
```

Comment:
This is for dominant color though, so it's not true.

Reply:
What do you mean it's not true? It's still a placeholder. The implementation details are different but I wouldn't think that should matter?

Reply:
```suggestion
      load a placeholder and not have to bother with creating a
```
Calling it an "image placeholder" makes it sound like the placeholder is an image rather than just a color, particularly with the bit about not needing to bother creating a thumbnail.

Reply:
Hmm ok. I can try to rephrase. I want to get across that it's a placeholder for an image that's of a given color. I don't think the underlying DOM structure is particularly relevant.

Reply:
Trying to work off the existing text was messing me up. I reworked the whole section.